encodedcc.py

```diff
@@ -662,8 +662,18 @@ def patch_set(args, connection):
                     if k[1] == "int" or k[1] == "integer":
                         patch_data[k[0]] = int(temp_data[key])
                     elif k[1] == "array" or k[1] == "list":
+                        if type(temp_data[key]) == str:
+                            # So JSON loads if present.
+                            temp_data[key] = temp_data[key].replace("'",'"')
```

Comment:
This line is strange.   So it accepts badly formed JSON as well?  Or is that just some excel work around?